Formula/rst-lint.rb

```diff
@@ -0,0 +1,39 @@
+class RstLint < Formula
+  include Language::Python::Virtualenv
+
+  desc "ReStructuredText linter"
+  homepage "https://github.com/twolfson/restructuredtext-lint"
+  url "https://github.com/twolfson/restructuredtext-lint/archive/1.1.2.tar.gz"
+  sha256 "baa99906eaafc00a975a8dee59f6bbbbecc21add2eb630dce6bef64ac0efd4d0"
+
+  head "https://github.com/twolfson/restructuredtext-lint.git", :shallow => false
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "docutils" do
+    url "https://files.pythonhosted.org/packages/84/f4/5771e41fdf52aabebbadecc9381d11dea0fa34e4759b4071244fa094804c/docutils-0.14.tar.gz"
+    sha256 "51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    # test invocation on a file with no issues
+    (testpath/"pass.rst").write <<~EOS
+    Hello World
```

Comment:
The audit failure message is pretty confusing but I'm pretty sure it means everything inside the `EOS` block should have a 2-space minimum indent.

Reply:
Aye. I'm not sure where the failure message is coming from, it doesn't appear to be hardcoded on the `brew` side so I'm presuming it's Rubocop being confusing.